sig/datadog/appsec/request_milestones.rbs

```diff
@@ -0,0 +1,10 @@
+module Datadog
+  module AppSec
+    class RequestMilestones
+      attr_accessor request_blocked: bool
```

Comment:
I think those name would change too because now we are already in the context of the request, and yet those metrics should be prefixed. If they will be guarded by `Context` class there will be no need to prefix them as the class name implies it.